Source/WebKit/Shared/DocumentEditingContext.mm

```diff
@@ -58,24 +82,38 @@ static inline NSRange toNSRange(DocumentEditingContext::Range range)
     }
 
     [platformContext setSelectedRangeInMarkedText:toNSRange(selectedRangeInMarkedText)];
+    setOptionalEditingContextProperties<UIWKDocumentContext>(*this, platformContext.get(), options);
 
-    for (const auto& rect : textRects)
-        [platformContext addTextRect:rect.rect forCharacterRange:toNSRange(rect.range)];
-
-    [platformContext setAnnotatedText:annotatedText.nsAttributedString().get()];
-
-#if HAVE(AUTOCORRECTION_ENHANCEMENTS)
-    if (options.contains(DocumentEditingContextRequest::Options::AutocorrectedRanges)) {
-        auto ranges = createNSArray(autocorrectedRanges, [&] (DocumentEditingContext::Range range) {
-            return [NSValue valueWithRange:toNSRange(range)];
-        });
-
-        if ([platformContext respondsToSelector:@selector(setAutocorrectedRanges:)])
-            [platformContext setAutocorrectedRanges:ranges.get()];
-    }
+    return platformContext.get();
```

Comment:
(Whoops — this is the cause for the mass test failures on iOS. Fixing in a moment)